MovieChecker/MovieChecker/BoxOfficeViewController.h

```diff
@@ -8,8 +8,8 @@
 
 #import <UIKit/UIKit.h>
 
-@interface ViewController : UIViewController
+@interface BoxOfficeViewController : UITableViewController
 
+@property (nonatomic, strong) NSMutableArray *movies;
```

Comment:
Conceptually, why expose this property?  Do any other classes need to know about it?  If so, why?